src/main/java/GeneralService/RequestHandlers/RequestHandler.java

```diff
@@ -0,0 +1,7 @@
+package GeneralService.RequestHandlers;
+
+import GeneralService.Request;
```

Comment:
deci cred ca am explicat si eu gresit, si dupa m-am pierdut si am mers pe ce ai facut... ca idee, 
1. sterge RequestHandler, 
2. ia metoda sendRequest din  WindowControlRequestHandler si pune-o in OnlineServicesCommunicationService 
3. sterge WindowControlRequestHandler 
4. rezolva restul de comentarii